File: tes-logika/src/countWords.js
function countWords(sentence) {
    const specialChars = ['*', '_', '!', '=', '[', ']', '(', ')', '{', '}', '&', '@', '#', '$', '%', '^', '+', '<', '>'];

    const words = sentence.split(' ');

    let count = 0;

    for (const word of words) {
        for (const char of word) {
            if (specialChars.includes(char)) {
                count++;
                break;
            }
        }
    }

    return words.length - count;
}

export default countWords

File: tes-logika-python/countWords.py
def count_words(sentence):
    special_chars = ['*', '_', '!', '=', '[', ']', '(', ')', '{', '}', '&', '@', '#', '$', '%', '^', '+', '<', '>']

    words = sentence.split()

    count = 0

    for word in words:
        for char in word:
            if char in special_chars:
                count += 1
                break

    return len(words) - count

def main():
    string = input('Masukkan kalimat, e.g "Kemarin Shopia per[gi ke mall.": ')

    print(count_words(string))
    
if __name__ == '__main__':
    main()